tests/phpunit/integration/Core/Consent_Mode/REST_Consent_Mode_ControllerTest.php

```diff
@@ -323,92 +323,92 @@ public function test_get_ads_measurement_status() {
 
 	public function test_get_ads_measurement_status__ads_module_connected() {
 		$this->setup_rest();
-		// Setup the site and admin user to make a successful REST request.
 		$this->grant_manage_options_permission();
 
-		$this->force_module_connection( Ads::MODULE_SLUG );
+		add_filter(
+			'googlesitekit_ads_measurement_connection_checks',
+			function () {
+				return array(
+					function () {
+						return true; },
```

Comment:
Not sure now PHPCS didn't flag this, but for simple one-liners you can use the newer `fn () => true` syntax.